React/Fabric/Mounting/ComponentViews/Switch/RCTSwitchComponentView.mm

```diff
@@ -34,9 +34,16 @@ - (instancetype)initWithFrame:(CGRect)frame
 
     _switchView = [[RCTUISwitch alloc] initWithFrame:self.bounds]; // [macOS]
 
+#if !TARGET_OS_OSX // [macOS]
     [_switchView addTarget:self action:@selector(onChange:) forControlEvents:UIControlEventValueChanged];
+#endif // [macOS]
```

Comment:
Can we add this method to `RCTUISwitch` ?

Reply:
I see that in `RCTSwitchManager.m` we have the following ifdef:

```
#if !TARGET_OS_OSX // [macOS]
  [switcher addTarget:self action:@selector(onChange:) forControlEvents:UIControlEventValueChanged];
#else // [macOS
  [switcher setTarget:self];
  [switcher setAction:@selector(onChange:)];
#endif // macOS]
```

If not implement the method, can we add the `#else` block?

Reply:
Resolved in other thread. This will be a followup.